packages/js/src/settings/routes/site-features.js

```diff
@@ -57,7 +58,7 @@ const FeatureCard = ( {
 						shouldDimHeaderImage && "yst-opacity-50 yst-filter yst-grayscale"
 					) }
 					src={ imageSrc }
-					alt={ imageAlt }
+					alt={ imageAlt ? imageAlt : "" }
```

Comment:
Not really needed, but could write it as:
```suggestion
					alt={ imageAlt ?? "" }
```
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing